Analysis/Utkscan/processors/include/PspmtProcessor.hpp

```diff
@@ -1,38 +1,122 @@
-/** \file PspmtProcessor.hpp
- *  \brief A processor to handle pixelated PMTs
- *  \author Shintaro Go
- *  \date November 16, 2015
- */
-
+///@file PspmtProcessor.cpp
+///@brief Processes information from a Position Sensitive PMT.
+///@authors S. Go and S. V. Paulauskas
```

Comment:
@akeeler : make sure you add your name to this list. You've contributed more to the file than just style changes.